packages/docs/src/styles/_sections.scss

```diff
@@ -260,6 +260,7 @@
   .pt-dialog {
     top: $content-padding;
     transform: translateX(50%);
+    z-index: inherit;
```

Comment:
`inherit` is a weird value to use here. 
i would prefer more explicit `initial` or `0` or `$pt-z-index-base` (which is 0).